components/channel_members_rhs/channel_members_rhs.tsx

```diff
@@ -122,6 +162,28 @@ export default function ChannelMembersRHS({channel, searchTerms, membersCount, c
         await actions.closeRightHandSide();
     };
 
+    const loadMore = async () => {
+        const nextPage = page + 1;
+        setPage(nextPage);
+        setLoadingMore(true);
+
+        await actions.loadProfilesAndReloadChannelMembers(nextPage, USERS_PER_PAGE, channel.id);
+        await actions.loadMyChannelMemberAndRole(channel.id);
```

Comment:
Similarly, why do we need to reload the current user's channel member each time?